api/base/generic_bulk_views.py

```diff
@@ -115,6 +125,18 @@ def bulk_destroy(self, request, *args, **kwargs):
         if not self.allow_bulk_destroy_resources(user, resource_object_list):
             raise PermissionDenied
 
-        self.perform_bulk_destroy(resource_object_list)
+        # If skip_uneditable=True in query_params, skip resources for which the user does not have admin permissions
+        # and delete the remaining resources
+        if self.request.query_params.get('skip_uneditable', 'False') == 'True':
```

Comment:
See other comment about using `is_truthy()`. There are a couple other of these instances in this PR, but I'm not going to flood your inbox with notifications for them.